amethyst_core/src/transform/components/transform.rs

```diff
@@ -585,6 +585,15 @@ impl Transform {
 
         res
     }
+
+    /// This is only exposed for debug builds, with `debug_assertions` set.
+    /// This function allows for test cases of copying the local matrix to the global matrix.
+    /// Useful for tests or other debug type access.
+    #[cfg(debug_assertions)]
```

Comment:
Why not just use `cfg(test)` then?

Reply:
I, at least, use it in actual debug builds too (moving objects around without the transform system). I thought it was prudent others might.

Reply:
If you think this might see external use then we should probably just include it unconditionally. What would be the issue with that?

Reply:
I guess I just feel changing the API surface area based on the optimization level chosen is arbitrary, and quite possibly frustrating for other users down the line.

Reply:
Ah, understood. I've now exposed it just as a regular API now.